src/cc65/declare.c

```diff
@@ -689,6 +689,13 @@ static int ParseFieldWidth (Declaration* Decl)
         Error ("Bit-field has invalid type '%s'", GetBasicTypeName (Decl->Type));
         return -1;
     }
+
+    if (SizeOf (Decl->Type) != SizeOf (type_uint)) {
+        /* Only int sized types may be used for bit-fields for now */
+        Error ("CC65 currently only supports unsigned int bit-fields");
```

Comment:
The name of this project is 'cc65', not 'CC65'.

Reply:
Will fix later.

Reply:
8eab280 already fixed it.